frontend/package.json

```diff
@@ -35,13 +42,14 @@
     "eslint-plugin-import": "2.25.4",
     "htmlhint": "1.1.4",
     "mdn-polyfills": "5.20.0",
+    "npm-run-all": "4.1.5",
     "parcel": "2.7.0",
     "postcss": "8.4.14",
     "postcss-mixins": "9.0.3",
     "posthtml-expressions": "1.9.0",
-    "posthtml-extend": "0.6.3",
     "prettier": "2.7.1",
     "prettier-plugin-tailwindcss": "0.1.11",
+    "rimraf": "3.0.2",
```

Comment:
I believe we don't use this package anymore.

Reply:
this is removed.